tests/lib/prepare-project.sh

```diff
@@ -119,18 +121,15 @@ if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
     quiet apt-get install -y --force-yes apparmor libapparmor1 seccomp libseccomp2 systemd cgroup-lite util-linux
 fi
 
-quiet apt-get purge -y snapd
+distro_purge_package snapd
```

Comment:
Those can get very long but they are immediately explicit and unambiguous. Anyone working on a given distribution can immediately relate to the outcome and predict what will happen. The special centralized identifiers (that sometimes match packages but not necessarily) feel a bit too magic to me. I specifically am not fond of the set of tables we need to maintain.

The `ID` and `ID_LIKE` fields in `/etc/os-release` can be used to key to filter the part before the colon. A single instruction:

`install_package debian:snapd`

Will work correctly on Debian and _all_ of its derivatives (through the `ID_LIKE` property).